### PR TITLE
Update the release notes with the 2.13.0 changelog

### PR DESCRIPTION
**Pull request by jnidzwetzki**
# Description

[Short summary of why you created this PR]

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: about/release-notes/index.md [hunk 1]
```diff
@@ -17,6 +17,76 @@ GitHub and be notified by email whenever a new release is available. On the
...
+This release contains performance improvements, an improved hypertable DDL API
+and bug fixes since the 2.12.2 release. We recommend that you upgrade at the next
+available opportunity.
+
+In addition, it includes these noteworthy features:
+
+* Full PostgreSQL 16 support for all existing features
+* Vectorized aggregation execution for sum()
+* Track chunk creation time used in retention/compression policies
+</Highlight>
```
Comment (iroussos) [inline about/release-notes/index.md]: The original version would highlight with a note the whole thing.. Also adding the noteworthy features under a "Highlighted features" sub-section

```suggestion
</Highlight>

#### Highlighted features in this release

This release includes these noteworthy features:

* Full PostgreSQL 16 support for all existing features
* Vectorized aggregation execution for sum()
* Track chunk creation time used in retention/compression policies
```

File: about/release-notes/index.md [hunk 2]
```diff
@@ -17,6 +17,76 @@ GitHub and be notified by email whenever a new release is available. On the
...
+
+#### Deprecation notice: Multi-node support
+TimescaleDB 2.13 is the last version that will include multi-node support. Multi-node
+support in 2.13 is available for PostgreSQL 13, 14 and 15. Learn more about it
+[here](docs/MultiNodeDeprecation.md).
```
Comment (iroussos) [inline about/release-notes/index.md]: This is a link towards our main repo, the file does not live in the docs site

```suggestion
[here](https://github.com/timescale/timescaledb/blob/main/docs/MultiNodeDeprecation.md).
```

**Review by iroussos — APPROVED** [review]
Thank you @jnidzwetzki, I added to small suggestions - other than that, we should be good to go on my side! Approving to let you merge at will

File: about/release-notes/index.md (hunk 1 above)
Comment (jnidzwetzki) [reply about/release-notes/index.md]: Changed

File: about/release-notes/index.md (hunk 2 above)
Comment (jnidzwetzki) [reply about/release-notes/index.md]: Good catch, changed

**Pull request merged**